src/pages/unsubscribeEmail.js

```diff
@@ -83,9 +79,21 @@ class UnsubscribeEmail extends React.Component {
           <Box my={3}>
             <Email size={42} color={this.getIconColor(this.state.state)} />
           </Box>
-          <MessageBox mb={3} type={this.state.state} withIcon>
-            <FormattedMessage {...this.getFormattedMessageAttrs(this.state.state)} />
-          </MessageBox>
+          {this.state.state === 'success' && (
+            <MessageBox mb={3} type="success" withIcon>
+              <FormattedMessage id="unsubscribe.success" defaultMessage="You've unsubscribed successfully !" />
+            </MessageBox>
+          )}
+          {this.state.state === 'unsubscribing' && (
+            <MessageBox mb={3} type="white" withIcon>
+              <FormattedMessage id="unsubscribe.unsubscribing" defaultMessage="Unsubscribing your email..." />
+            </MessageBox>
+          )}
+          {this.state.state === 'error' && (
+            <MessageBox mb={3} type="error" withIcon>
+              <FormattedMessage {...this.getErrorMessageAttrs(this.state.state)} />
```

Comment:
I would skip `FormattedMessage` here if the message can't be translated.

Reply:
@znarf this is the error message from the server

Reply:
Yes, my point is that we don't need to pass it trough `<FormattedMessage>` if we don't get anything out of that.

Reply:
oh I got it I replaced it with span tag.